test/alternator/test_returnvalues.py

```diff
@@ -441,3 +486,27 @@ def test_update_item_returnvalues_all_new_add_existing(test_table_s):
         UpdateExpression='ADD n :inc',
         ExpressionAttributeValues={':inc': 1})
     assert ret['Attributes']['n'] == 4
+
+# Testing feature which returns values only on modification condition failures.
```

Comment:
Again all my comments above about this comment. In particular, it's not a "modification condition failure", is a "UpdateItem with a failed condition expression".

Reply:
ok